src/request/iframe/IFrameApi.js

```diff
@@ -8,25 +8,41 @@
 class IFrameApi {
     /**
      * @param {Rpc.State | null} state
-     * @param {boolean} [listFromLegacyStore] - Deprecated, only for database migration
```

Comment:
I am not sure, but I'd personally rather have an additional function
```
/**javascript
 * @param {Rpc.State | null} state
 * @deprecated
 * @returns {Promise<KeyInfoObject[]>}
*/
async listFromLegacyStore(state) {
```
than the same function with different parameters being deprecated or not.
Also i think @deprecated should be added to deprecated methods, which also is not exactly possible with the deprecated and at the same time not deprecated methods.